dist/index.html

```diff
@@ -7,26 +7,27 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link href="./index.css">
     <title>Leaderboard</title>
-    <script defer src="/Leaderboard/main.js"></script>
-</head>
+<script defer src="main.js"></script></head>
 
 <body>
-    <h1>Leaderboard</h1>
-    <div class="flex items-center">
+    <div class="container">
         <div>
-            <div class="flex justify-between items-center">
-                <h2>Recent Scores</h2>
-                <button>Refresh</button>
+            <h1>Leaderboard</h1>
+            <div class="layout">
+                <div class="left">
+                    <div class="heading">
+                        <h2>Recent Scores</h2>
+                        <button id="refresh">Refresh</button>
+                    </div>
+                    <!-- table -->
+                </div>
+                <div class="right">
+                    <h2>Add your score</h2>
+                    <input required id="name" type="text" placeholder="Your Name"><br>
+                    <input required id="score" type="text" placeholder="Your Score"><br>
```

Comment:
- [ ] [Optional] [Highly recommended] Please Insure that the score input field takes only numbers because now it takes text too.

[Hint]: you can change type to number instead of text.